### PR TITLE
Open Telemetry End To End Test: Adds baseline for network level requests trace

### PR DESCRIPTION
**Pull request by sourabh1007**
## Description
This PR is doing below things:
1. Setting DisplayName for an operation level activity as `<operationname><space><containername>`
2.  Enabling Network level tracing **`EndtoEndBaselineTests`**

### CODE REVIEW

**Review by ealsur — APPROVED** [review]

File: Microsoft.Azure.Cosmos/src/Telemetry/OpenTelemetry/OpenTelemetryRecorderFactory.cs [hunk 1]
```diff
@@ -51,6 +51,8 @@ internal static class OpenTelemetryRecorderFactory
                 // Record values only when we have a valid Diagnostic Scope
                 if (scope.IsEnabled)
                 {
+                    scope.SetDisplayName($"{operationName} {containerName}.{databaseName}");
```
Comment (jcocchi) [inline Microsoft.Azure.Cosmos/src/Telemetry/OpenTelemetry/OpenTelemetryRecorderFactory.cs:54]: I thought we decided to use <db.operation> <db.cosmosdb.container>

**Comment by ealsur** [comment]
@msftbot wait for @jcocchi to approve

File: Microsoft.Azure.Cosmos/src/Telemetry/OpenTelemetry/OpenTelemetryRecorderFactory.cs (hunk 1 above)
Comment (sourabh1007) [reply Microsoft.Azure.Cosmos/src/Telemetry/OpenTelemetry/OpenTelemetryRecorderFactory.cs:54]: Sorry I misread old conversation. :(

**Review by ealsur — COMMENTED** [review]
Holding off for Justine's comment

**Review by jcocchi — APPROVED** [review]

**Pull request merged**